apps/android/app/src/main/java/chat/simplex/app/SimplexApp.kt

```diff
@@ -69,9 +69,9 @@ class SimplexApp: Application(), LifecycleEventObserver {
     withApi {
       when (event) {
         Lifecycle.Event.ON_STOP ->
-          if (!appPreferences.runServiceInBackground.get()) SimplexService.stop(applicationContext)
+          if (appPreferences.notificationsMode.get() != NotificationsMode.SERVICE.name) SimplexService.stop(applicationContext)
         Lifecycle.Event.ON_START ->
-          if (chatModel.chatRunning.value != false)  SimplexService.start(applicationContext)
+          if (chatModel.chatRunning.value != false) SimplexService.start(applicationContext)
```

Comment:
I think it should also depend on `appPreferences.notificationsMode.get()` - if the notification mode is "periodic" then this call shouldn't happen, or am I wrong?